pkg/webconfig/tls_credentials.go

```diff
@@ -209,17 +209,17 @@ func (tr *tlsReferences) getCertFilename() string {
 	return ""
 }
 
-// getCAMountPath is the mount path of the client CA certificate inside a container.
-func (tr *tlsReferences) getCAMountPath() string {
+// getCAMountPath is the mount path of the client CA certificate inside a prometheus container.
+func (tr *TLSReferences) GetCAMountPath() string {
 	if tr.clientCA.ConfigMap != nil || tr.clientCA.Secret != nil {
 		return tr.tlsPathForSelector(tr.clientCA, "ca")
 	}
 
 	return ""
 }
 
-// getCAFilename retruns the filename (key) of the client CA certificate.
-func (tr *tlsReferences) getCAFilename() string {
+// getCAFilename is the mount path of the client CA certificate inside a prometheus container.
```

Comment:
hmm, the comment doesn't match what the function does?

Reply:
Yep, must be a mistake.